app/tests/base.py

```diff
@@ -25,15 +25,6 @@ def setUp(self):
             self.short_homework_quiz_data
         )
 
-        # long assessment quiz
-        self.long_assessment_quiz_data = json.load(
-            open("app/tests/dummy_data/long_assessment_quiz.json")
-        )
-        (
-            self.long_assessment_quiz_id,
-            self.long_assessment_quiz,
-        ) = self.post_and_get_quiz(self.long_assessment_quiz_data)
-
```

Comment:
why did you remove this? was this test data updated with multi question set data?

Reply:
yes, whatever tests we were making with long assessment, we can do the same with multi question set quiz. hence, i removed it. i've retained the long assessment json file though.